packages/components/dropdown/src/dropdown.ts

```diff
@@ -87,7 +87,16 @@ export const dropdownProps = buildProps({
   buttonProps: {
     type: definePropType<ButtonProps>(Object),
   },
+  showArrow: {
+    type: Boolean,
+    default: true,
+  },
+  offset: {
+    type: Number,
+    default: undefined,
```

Comment:
What is written in the document is `0`, and it should also be `0` here.

Reply:
yeah, The new props are all moved from the Popover, and I think the Popover has the same problem